content/en/docs/reference/glossary/cluster-operations.md

```diff
@@ -4,10 +4,17 @@ id: cluster-operations
 date: 2019-05-12
 full_link:
 short_description: >
- Activities such as upgrading the clusters, implementing security, storage, ingress, networking, logging and monitoring, and other operations involved in managing a Kubernetes cluster.
+ The work involved in managing a Kubernetes cluster.
 
 aka:
 tags:
-- operations
+- operation
 ---
- Activities such as upgrading the clusters, implementing security, storage, ingress, networking, logging and monitoring, and other operations involved in managing a Kubernetes cluster.
+ The work involved in managing a Kubernetes cluster.
+
+<!--more-->
+
+ Examples of cluster operations work include: deploying new Nodes to
+scale the cluster; performing software upgrades; installing security
```

Comment:
@sftim , nit: The original definition divides the work into:
 upgrading clusters (software and hardware) and administering (managing) clusters.
I think implementing logging and monitoring is different from `responding to events`.
Is `implementing security` the same as `installing security`. What do you think?

Reply:
I'll bear that in mind!